config/initializers/paperclip.rb

```diff
@@ -19,12 +19,7 @@
 class PaperclipSettings
 
   def self.config
-    secrets = Hash(Rails.application.secrets.paperclip).symbolize_keys
-    Settings.paperclip.to_hash.merge(secrets)
-  end
-
-  def self.fog?
-    config[:storage] == "fog"
+    Settings.paperclip.to_hash
```

Comment:
At some point we should consider getting rid of this `PaperclipSettings` class altogether now that it's not doing much anymore.